### DISCUSSION TITLE
Daily 30/9

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#45] [General]
Como es feriado en Córdoba es opcional para los cordobeses. Hoy voy a hacer un repaso general de HTML y CSS para ver si puedo hacer el examen hoy o mañana, también me gustaría seguir convirtiendo las historias de usuario en tareas. Como no hay clases de full-stack ayer vi una parte de la del jueves, voy a ver si puedo ver el resto hoy.

**Comment by NataliaAlvarezIspc** [comment]
Ayer participè de las clases sincrónicas de SGC y de Full Stack. Estuve repasando CSS.
Hoy tuve clase sincróncica de Inglés. Voy a estar repasando los contenidos de Full Stack para el exámen y continuaré con Python en la plataforma Cisco

**Comment by claubott** [comment]
No escribí en un par de de días, pero hice la actividad 2 de SGC, estuve avanzando con poo y cisco y hoy rendí front-end.

**Comment by Brian-Almada** [comment]
Ayer no pude realizar nada porque me ocupé de unas tareas fuera del estudio. Ahora estoy repasando para el examen de FS y probando código en Python para repasar y practicar.
